main.js

```diff
@@ -0,0 +1,59 @@
+const {app, BrowserWindow} = require('electron')
+const path = require('path')
+const url = require('url')
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let win
+
+function createWindow () {
+  // Create the browser window.
+  win = new BrowserWindow({
+    width: 1920,
+    height: 1080,
+    icon: path.join(__dirname, 'oh_noes!')
```

Comment:
If you find a cool icon, you can reference it here and it will show up in the task menu, dock, or whatever the DE uses